generator/src/test/java/com/scottlogic/deg/generator/constraints/IsEqualToConstantConstraintTest.java

```diff
@@ -41,6 +41,19 @@ public void testConstraintIsNotEqualDueToValue() {
         Assert.assertNotEquals(constraint1, constraint2);
     }
 
+    @Test
+    public void testConstraintThrowsIfGivenNull(){
```

Comment:
As in IsInSetConstraintTests  - could we rewrite it so it is easier to follow what we are asserting?

Reply:
Changed